lib/jekyll/theme_builder.rb

```diff
@@ -56,12 +56,27 @@ def write_file(filename, contents)
 
   def create_directories
     mkdir_p(SCAFFOLD_DIRECTORIES)
+    mkdir_p("_sass/#{name}")
   end
 
   def create_starter_files
     %w(page post default).each do |layout|
       write_file("_layouts/#{layout}.html", template("_layouts/#{layout}.html"))
     end
+
+    %w(footer head header).each do |incl|
```

Comment:
@parkr I've always used `head`, `header` and `footer` since they match the HTML elements. I'm sure most who write themes for Jekyll will recognise this.